fabric/src/main/java/com/discordsrv/fabric/FabricAdventureUtil.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * This file is part of DiscordSRV, licensed under the GPLv3 License
+ * Copyright (c) 2016-2025 Austin "Scarsz" Shapiro, Henri "Vankka" Schubin and DiscordSRV contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.discordsrv.fabric;
+
+import com.discordsrv.api.component.MinecraftComponent;
+import com.discordsrv.common.util.ComponentUtil;
+import net.kyori.adventure.audience.Audience;
+import net.kyori.adventure.text.Component;
+import net.minecraft.server.command.CommandOutput;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.text.Text;
+import org.jetbrains.annotations.NotNull;
+
+//? if adventure: <6 {
+/*import net.kyori.adventure.platform.fabric.FabricServerAudiences;
+import net.kyori.adventure.platform.fabric.AdventureCommandSourceStack;
+ *///?} else {
+import net.kyori.adventure.platform.modcommon.MinecraftServerAudiences;
+import net.kyori.adventure.platform.modcommon.AdventureCommandSourceStack;
+//?}
+
+public class FabricAdventureUtil {
```

Comment:
The `Util` suffix is used for classes with only static methods. For consistency sake I would prefer a different name. 
Some possibilities are: just `FabricAdventure`, or `FabricComponentFactory` extending `ComponentFactory` (`DiscordSRV#componentFactory`)